packages/order/integration-tests/__tests__/create-order.ts

```diff
@@ -194,7 +194,8 @@ moduleIntegrationTestRunner({
       it("should create an order, shipping method and items. Including taxes and adjustments associated with them", async function () {
         const createdOrder = await service.create(input)
 
-        expect(createdOrder).toEqual(expectation)
+        const serializedOrder = JSON.parse(JSON.stringify(createdOrder))
```

Comment:
Q: shouldnt it be already serialized when the service return the order?

Reply:
this "serialization" is just to transform the BigNumbers to numbers. the serialize of the Mikro-ORM entity already happened inside the module